packages/docs/src/styles/components/tableOfContents.scss

```diff
@@ -0,0 +1,25 @@
+.c-table-of-contents {
+  position: sticky;
+  top: $spacer-1;
+}
+
+.c-table-of-contents__heading {
+  border-bottom: 1px solid $color-gray-lighter;
+  padding-bottom: 0.25rem;
+}
+
+.c-table-of-contents__list {
+  list-style-type: '- ';
+  margin-top: $spacer-half;
```

Comment:
Looking at the designs, it looks more like a `$spacer-1` to me. I tagged @line47 as a reviewer though.

Reply:
Agreed, `$spacer-1` would work well here